db/1.12/modules/ROOT/pages/database-execute-bulk.adoc

```diff
@@ -0,0 +1,245 @@
+= Execute Bulk Operations Examples - Mule 4
+
+Anypoint Connector for Database (Database Connector) supports the *Bulk insert*, *Bulk update*, and *Bulk delete* operations that enables you to manage data in bulk. This improves the performance compared to executing a single insert, update or delete operation at various times.
+
+These operations are similar as their single counterparts, except that instead of receiving input parameters as key-value pairs, the bulk operations expect a list of key-value pairs. Avoid unnecessary steps by doing a bulk operation:
```

Comment:
```suggestion
These operations are similar to their single counterparts, except that instead of receiving input parameters as key-value pairs, the bulk operations expect a list of key-value pairs. Avoid unnecessary steps by doing a bulk operation:
```